expo/gbe/models/remains.py

```diff
@@ -483,13 +483,29 @@ def append_alerts(self, alerts):
         '''
         return alerts
 
-    @property
-    def promo_small(self):
+    '''
+    gets the URL for images that have been sized specifically to
+    suit GBE layouts
+       mini = the size for show/act and fashion fair grids - height = 125p
+       thumb = the size for the /gbe page -  height 75-100p
+    There is a timebased script that minifies uploaded images.  When using
+    this function, always check first that the image exists, and if it
```

Comment:
Shouldn't this function be responsible for this check, if it's needed every time it's called?